.changes/1095.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "fixed",
```

Comment:
I've changed it so this used a fixed entry while the other uses a changed entry, so this makes it clear that this is a bug fix, and the other is a part of the image upgrade.